lib/src/render/documentation_renderer.dart

```diff
@@ -68,3 +76,256 @@ class DocumentationRenderResult {
   const DocumentationRenderResult(
       {@required this.asHtml, @required this.asOneLiner});
 }
+
+bool _allowClassName(String className) =>
+    className == 'deprecated' || className.startsWith('language-');
+
+Iterable<String> _addLinkRel(String uri) {
+  final u = Uri.tryParse(uri);
+  if (u == null || !['http', 'https', 'mailto'].contains(u.scheme)) {
+    return ['nofollow'];
```

Comment:
Hmm.  This seems to add nofollow to comment references, not just random links.   I could see an argument for random links, but why comment references?  Telling Google to ignore them is probably going to degrade search rather than improve it on most documentation sets.